spec/models/comment_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
```

Comment:
- [ ] Kindly fix the syntax error in `comment_spec.rb` as this is causing a failure of all tests to run.
![image](https://user-images.githubusercontent.com/81932478/195163771-01c272d3-4946-4da5-a1d2-5a69f18163a7.png)